### ISSUE TITLE
Slight tweak suggestions: Show cost of funding awards

### ISSUE THREAD

**Issue by CosmoCora** [terraforming-mars/terraforming-mars#3729]
This is a really great game implementation. A very small thing that would be great to add is showing the cost of funding awards when selecting "Fund Award" ... It's easy to remember the first award costs 8 MC, but the 14 MC and 20 MC costs are a bit harder to remember.

**Comment by kberg** [comment]
Hi -- if you enable Learner Mode in the preferences, they'll appear on the awards:

![image](https://user-images.githubusercontent.com/413481/137659085-438ee255-3256-468f-af69-6d19c5f08177.png)
![image](https://user-images.githubusercontent.com/413481/137659131-54446251-c385-458f-b1db-ab8a8afe8f29.png)

Is that sufficient for you?

**Comment by bafolts** [comment]
@kberg which options buttons would show the price? If you include screenshot I could put up the change. it makes sense to me as well to display this value.

**Comment by kberg** [comment]
@bafolts like so: 
![image](https://user-images.githubusercontent.com/413481/138008368-e007a94c-6fee-4a0f-8a6c-bd78c51e4c9c.png)

**Comment by kberg** [comment]
I agree completely. I would like to turn on a default for all new servers,
but have not dug into how to tell whether this is a fresh install.

>
